.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: php
+php:
+  - 7.1
+  - 7.2
+  - 7.3
+  - 7.4
+  - 5.4 #Drop it??
+  - 5.6
+services: mysql # Credentials for mysql based on the documentation https://docs.travis-ci.com/user/database-setup/#mysql
+before_script:
+  - git clone --depth=50 --branch=master https://shortpixel-travis-user:SP_Password_2019@github.com/short-pixel-optimizer/plugin-modules.git modules 
```

Comment:
Let's change the authentication to use a Token or a more secure way than this.

Reply:
Solved. Password changed, token uploaded